### ISSUE TITLE
Prefer structured communication

### ISSUE THREAD

**Issue by Thrashattack** [GeekHunter-Brasil/hello-world#16]
As long as the charpter is growing and the number of people is increasing, we will have less and less oportunities to sync the whole charpter. 
Due to that, whenever you feel the need to start a discussion or anything similar, prefer structured communications like Github Issues instead Slack. It will make far more easy to newcomers to reach this topics and issues that may, in sometime, be their issues. 
It also improves a lot our documentation about our own development as a reference software engineer team.

**Comment by paulohenriquepm** [comment]
I agree! Keeping a history of our discussions it's also a good way to remember why we decided or not to do something.

**Comment by Thrashattack** [comment]
> Agreed 100%, it's also easier to center everything in one place and keep things organized.
> 
> We can also have more styleguides and documents, other than backend and frontend (e.g. development styleguide, code review styleguide, etc)

warming up my neurons here to make a PR creating the development styleguide o/
